openverse_catalog/dags/providers/provider_api_scripts/finnish_museums.py

```diff
@@ -127,6 +127,18 @@ def _process_object(obj, sub_providers=SUB_PROVIDERS, provider=PROVIDER):
     return total_images
 
 
+def get_license_url(obj):
```

Comment:
Do you mind adding unit tests for this function as well? Or do you feel like it's sufficiently encapsulated in the current testing examples?

Reply:
I think this is covered in the current example -- we simulate receiving [this example json](https://github.com/WordPress/openverse-catalog/blob/main/tests/dags/providers/provider_api_scripts/resources/finnishmuseums/object_complete_example.json#L28) from the API, which includes the `/deed.fi` suffix on the license url, and then assert that the item added to the ImageStore has the [correctly formatted url](https://github.com/WordPress/openverse-catalog/pull/418/files#diff-17dd91ce4ca74ccedfbf229306a08034f674305c6b5dacbebe8bfc6741728a37R100). Although maybe I could add an additional test to verify it still works if the API returns urls without that suffix